src/QuartzNET-DynamoDB.Tests/Integration/JobStore/JobResumeTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Quartz.DynamoDB.Tests.Integration;
+using Quartz.Simpl;
+using Quartz.Spi;
+using Xunit;
+
+namespace Quartz.DynamoDB.Tests
```

Comment:
Nice spot, somehow I have the namespace of all of these files incorrect... Will fix.